examples/js/media/src/widgets/Dates.ts

```diff
@@ -1,8 +1,9 @@
 import { numericMenu, panel } from 'instantsearch.js/es/widgets';
 
 import { DATE_RANGES } from '../utils';
+import { PanelTemplates } from 'instantsearch.js/es/widgets/panel/panel';
 
-const createDatesList = (header: string) =>
+const createDatesList = (header: PanelTemplates<any>['header']) =>
```

Comment:
```suggestion
const createDatesList = (header: PanelTemplates<typeof numericMenu>['header']) =>
```

not sure if that's correct, but I believe the type can be found